backend/forum/functions/createPost/handler.ts

```diff
@@ -1,7 +1,9 @@
+import { HandlerType } from '@swarmion/serverless-contracts';
+
 import { createPostContract } from '@swarmion-starter/forum-contracts';
 import { applyHttpMiddlewares } from '@swarmion-starter/serverless-helpers';
 
-export const handler = createPostContract.handler(async event => {
+export const handler: HandlerType<typeof createPostContract> = async event => {
```

Comment:
Here you can use the `getLambdaHandler` method https://www.swarmion.dev/docs/contracts/api-gateway#type-the-lambda-input-and-output